firefox-ios/Client/Frontend/Autofill/Address/Edit/UpdatableAddressFields+Decodable.swift

```diff
@@ -5,7 +5,7 @@
 import Storage
 import struct MozillaAppServices.UpdatableAddressFields
 
-extension UpdatableAddressFields: Decodable {
+extension UpdatableAddressFields: Swift.Decodable {
```

Comment:
Was there a domain/naming conflict here that required adding the `Swift.`?

Reply:
In `Xcode 16` - `Swift 6` we get an error if we conform to protocols in extensions. For example, in this case we get: `Extension declares a conformance of imported type 'UpdatableAddressFields' to imported protocol 'Decodable'; this will not behave correctly if the owners of 'MozillaAppServices' introduce this conformance in the future`. The compiler suggests to fix this by adding `@retroactive Decodable`, but `@retroactive` is available only in Swift 6, so we would not be able to compile our project for lower Swift versions. By adding `Swift.Decodable` we make both happy.

Reply:
Thank you, this was news to me